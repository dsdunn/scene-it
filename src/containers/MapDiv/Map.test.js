import React from 'react';
import { shallow, mount } from 'enzyme';
import { Map } from './index';

describe('MapDiv', () => {

  const mockProps = {
    center: {lat:1, lng: 2},
    events: [],
    selectedEvent: {some: 'event'}
  }

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Map {...mockProps}/>)
  })

  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})