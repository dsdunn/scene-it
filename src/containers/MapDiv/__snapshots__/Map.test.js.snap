// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapDiv matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Map
    center={
        Object {
            "lat": 1,
            "lng": 2,
          }
    }
    events={Array []}
    selectedEvent={
        Object {
            "some": "event",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "center": Object {
        "lat": 1,
        "lng": 2,
      },
      "children": Array [
        <MarkerClusterer
          averageCenter={true}
          enableRetinaIcons={true}
          gridSize={20}
          zoomOnClick={true}
>
          
</MarkerClusterer>,
        <InfoWindow
          position={
                    Object {
                              "lat": undefined,
                              "lng": undefined,
                            }
          }
>
          <p
                    className="infoText"
          />
</InfoWindow>,
      ],
      "zoom": 10,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "averageCenter": true,
          "children": Array [],
          "enableRetinaIcons": true,
          "gridSize": 20,
          "zoomOnClick": true,
        },
        "ref": null,
        "rendered": Array [],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <p
            className="infoText"
/>,
          "position": Object {
            "lat": undefined,
            "lng": undefined,
          },
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "infoText",
          },
          "ref": null,
          "rendered": null,
          "type": "p",
        },
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "center": Object {
          "lat": 1,
          "lng": 2,
        },
        "children": Array [
          <MarkerClusterer
            averageCenter={true}
            enableRetinaIcons={true}
            gridSize={20}
            zoomOnClick={true}
>
            
</MarkerClusterer>,
          <InfoWindow
            position={
                        Object {
                                    "lat": undefined,
                                    "lng": undefined,
                                  }
            }
>
            <p
                        className="infoText"
            />
</InfoWindow>,
        ],
        "zoom": 10,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "averageCenter": true,
            "children": Array [],
            "enableRetinaIcons": true,
            "gridSize": 20,
            "zoomOnClick": true,
          },
          "ref": null,
          "rendered": Array [],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <p
              className="infoText"
/>,
            "position": Object {
              "lat": undefined,
              "lng": undefined,
            },
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": undefined,
              "className": "infoText",
            },
            "ref": null,
            "rendered": null,
            "type": "p",
          },
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
